tests/regression/facebook/test_chrome_facebook_ail_type_comment_1_txt.sikuli/test_chrome_facebook_ail_type_comment_1_txt.py

```diff
@@ -45,8 +45,7 @@ def run(self):
 
         # Record T1, and capture the snapshot image
         # Input Latency Action
-        screenshot, t1 = fb.il_type('a', capture_width, capture_height,
-                                    wait_component=fb.FACEBOOK_POST_ACTION)
+        screenshot, t1 = fb.il_type('a', capture_width, capture_height)
```

Comment:
It should be fine since mouse delay is 0, and the click should be immediately focus on fb comment input box. Thanks.